backend/Pipfile

```diff
@@ -1,27 +0,0 @@
-[[source]]
```

Comment:
I will be better we continue using Pipfile over requirements.txt; https://jonathanmeier.io/advantages-of-pipfile-lock-over-requirements-txt/

Reply:
I had initially gone with Pipfile because of automatic dependency management and all the good stuff that pipefile comes with, however it seems like there are some dependencies especially with flask views that dont go well with Pipefile, infact I lost two days thinking I had written bad code only to realize that using Pipfile was updating some dependencies causing incompatibility with others. This led me to switch back to requirements.txt, we can however have a session where I can show you the issue then we can debug and find a solution and switch back to using pipfile

Reply:
sounds good